packages/zcli-apps/src/commands/apps/server.ts

```diff
@@ -53,14 +54,39 @@ export default class Server extends Command {
       res.end(JSON.stringify(appJSON))
     })
 
-    appJSON.installations.forEach((installation: Installation, index: number) => {
-      app.use(`/${installation.app_id}/assets`, express.static(`${appPaths[index]}/assets`))
-    })
+    const setAppAssetsMiddleware = () => {
+      appJSON.installations.forEach((installation: Installation, index: number) => {
+        app.use(`/${installation.app_id}/assets`, express.static(`${appPaths[index]}/assets`))
+      })
+    }
+
+    // Keep references of watchers for unwatching later
+    const watchers = appPaths.map(appPath =>
+      fs.watch(appPath, async (eventType, filename) => {
+        if (filename.toLowerCase() === 'manifest.json') {
+          // Regenerate app.json
+          appJSON = await buildAppJSON(appPaths, port)
+          // Reset middlewares for app assets
+          setAppAssetsMiddleware()
+          this.log('manifest.json changed and please refresh.')
```

Comment:
Maybe reword this slightly to `The manifest of the served app has changed, please refresh in your preferred host (eg Support or Sell) if you wish to pick up the changes.`

Reply:
Yup, I just feel that it's a little bit long. I've asked @zach-anthony to see if he has any preferred wording.

Reply:
removed the log as per @zach-anthony suggested.